plugins/content/transcript/lib/api/KalturaTranscriptAsset.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @package plugins.transcript
+ * @subpackage api.objects
+ */
+class KalturaTranscriptAsset extends KalturaAttachmentAsset 
+{
+	/**
+	 * The accuracy of the transcript
+	 * @var float
+	 */
+	public $accuracy;
+
+	/**
+	 * Was verified by human or machine
+	 * @var bool
+	 */
+	public $humanVerified;
```

Comment:
Make it nullable boolean or make it readonly, it doesn't make sense to enforce setting it on every update.